kong-control-plane-cassandra.yaml

```diff
@@ -75,6 +75,15 @@ spec:
           - name: KONG_CASSANDRA_KEYSPACE
             value: kong
         command: [ "kong", "migrations", "bootstrap" ]
+      - name: kong-migration-up
+        image: kong
+        imagePullPolicy: IfNotPresent
+        env:
+          - name: KONG_PG_PASSWORD
+            value: kong
+          - name: KONG_PG_HOST
+            value: postgres.kong.svc
+        command: [ "kong", "migrations", "up" ]
```

Comment:
This is not sufficient. You need to do `kong migrations finish` as well.
Also, we should do this using a `Job` and not as an init-container.

Reply:
does doing this as a `Job` apply to bootstrap as well?

Reply:
Yes. Please have a look at our Helm chart on how we are doing it:
https://github.com/helm/charts/tree/master/stable/kong/templates

We do a pre-migration and a post-migration for this purpose.

Reply:
done in https://github.com/Kong/kong-dist-kubernetes/pull/97